tests/test_validate_data.py

```
@@ -0,0 +1,215 @@
+        self.temp_dir = tempfile.mkdtemp()
+        self.data_dir = self.project_root / "data" / "raw"
+
+    def tearDown(self):
+        """テスト後処理: 一時ディレクトリをクリーンアップ"""
+        shutil.rmtree(self.temp_dir, ignore_errors=True)
+
+    def test_format_option_json(self):
+        """JSON形式出力のテスト"""
+        output_file = Path(self.temp_dir) / "report.json"
+
+        result = subprocess.run(
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)